build.sbt

```diff
@@ -65,11 +65,11 @@ lazy val kafka =
       resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
       libraryDependencies ++= Seq(
         "dev.zio"                   %% "zio-streams"             % zioVersion,
-        "dev.zio"                   %% "zio-test"                % zioVersion % "test",
-        "dev.zio"                   %% "zio-test-sbt"            % zioVersion % "test",
+        "dev.zio"                   %% "zio-test"                % zioVersion % Test,
+        "dev.zio"                   %% "zio-test-sbt"            % zioVersion % Test,
         "org.apache.kafka"           % "kafka-clients"           % kafkaVersion,
-        "com.fasterxml.jackson.core" % "jackson-databind"        % "2.13.1",
```

Comment:
This update to v2.13.* was a mistake I made. the 2.13.* version of jackson are the "unstable" ones